.github/workflows/test.yml

```
@@ -172,10 +187,20 @@ jobs:
       - name: Verify basic
         run: __test__/verify-basic.sh --archive
 
+  inspect:
+    runs-on: ubuntu-latest
+    steps:
+      - run:  ls -all /usr/bin
+
   test-proxy:
     runs-on: ubuntu-latest
     container:
-      image: alpine/git:latest
+      image: ubuntu:latest
+      volumes:
+        - /usr/bin/git:/mapped/bin/git
+        - /usr/bin/git-lfs:/mapped/bin/git-lfs
+        - /usr/bin/git-ftp:/mapped/bin/git-ftp
+        - /usr/bin/git-shell:/mapped/bin/git-shell
```

After lots of experimentation, it's clear that (as @joshmgross suggested) a custom image hosted on GitHub is going to be the easiest (most maintainable) path forward, here.

```suggestion
      image: ghcr.io/actions/ubuntu-with-git:latest
```